### PR TITLE
fix labels styling on VM details page

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

fixing stying to labels component on VM details page

## 🎥 Demo

fix-labels-css

Signed-off-by: Aviv Turgeman <aturgema@redhat.com>

### CODE REVIEW

File: src/views/virtualmachines/details/tabs/details/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.vm-description-item__title {
+    justify-content: space-between;
+    flex: 1;
+  }
+
```
Comment (glekner) [inline src/views/virtualmachines/details/tabs/details/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem.scss:5]: prettier

File: src/views/virtualmachines/details/tabs/details/components/VirtualMachineLabels/VirtualMachineLabels.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.vm-labels-group {
+    padding-top: var(--pf-c-label-group--m-category--PaddingTop);
+    padding-right: var(--pf-c-label-group--m-category--PaddingRight);
+    padding-bottom: var(--pf-c-label-group--m-category--PaddingBottom);
+    padding-left: var(--pf-c-label-group--m-category--PaddingLeft);
+    border: var(--pf-c-label-group--m-category--BorderWidth) solid
+      var(--pf-c-label-group--m-category--BorderColor);
+  }
+
```
Comment (glekner) [inline src/views/virtualmachines/details/tabs/details/components/VirtualMachineLabels/VirtualMachineLabels.scss:9]: prettier

File: src/views/virtualmachines/details/tabs/details/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem.scss (hunk 1 above)
Comment (avivtur) [reply src/views/virtualmachines/details/tabs/details/components/VirtualMachineDescriptionItem/VirtualMachineDescriptionItem.scss]: good catch

**Pull request merged**